common/src/test/java/tachyon/util/FormatUtilsTest.java

```diff
@@ -95,6 +97,32 @@ public TestCase(String expected, ByteBuffer input) {
   }
 
   /**
+   * Tests the {@link FormatUtils#byteArrayToHexString(byte[])} method.
+   */
+  @Test
+  public void byteArrayToHexStringTest() {
+    class TestCase {
```

Comment:
I agree. with < 10 test cases, I am fine with each one having a separate assert as it is really easy to read and reason.